module/project/ProjectDialog.kt

```diff
@@ -149,43 +149,19 @@ object ProjectDialog {
             )
             Form.IconButton(
                 icon = Icon.FOLDER_OPEN,
-                tooltip = Tooltip.Arg(Label.OPEN_PROJECT_DIRECTORY)
-            ) { launchDirectorySelector(state, window, title) }
+                tooltip = Tooltip.Arg(title)
+            ) {
+                val selectedDirectoryPath = SelectFileDialog.selectPath(
```

Comment:
Hmm, the problem with `selectPath` is it sounds like an action that the function itself takes - ie: you call this method, it selects a path for you. Whereas `SelectFileDialog.open` is clear: it opens the dialog, and then the user is free to play around in it.

However I do see the awkwardness of a method named `open` just returning a `String?`. Let's introduce a thin wrapper class just to clarify what the method does, and returns:
```kt
class SelectFileDialog {
    data class Result(val selectedPath: String?)
}
```
Now the signature of `open` makes sense, it's clear what it does, and the usage API is clean - the current line can in fact (I believe) remain almost identical to what it is now by using destructuring:
```suggestion
                val (selectedPath) = SelectFileDialog.open(
```